### PR TITLE
add bitcoin da config for backup directory

### LINKED ISSUE #1147: Configurable directory for inscriptions txs
Currently, sending sequencer commitment fails because the inscriptions tx directory doesn't exist and/or we're using MANIFEST_DIR. This causes the sending to fail:
```
{“timestamp”:“2024-09-10T07:44:33.536862Z”,“level”:“ERROR”,“fields”:{“message”:“Failed to send transaction to DA layer”,“e”:“No such file or directory (os error 2)“},“target”:“bitcoin_da::service”}
```

1. Make the directory in which we save those files configurable
2. Create directory if it does not exist.

### PR DESCRIPTION
**Pull request by eyusufatik**
# Description

## Linked Issues
closes #1147

## Testing
Describe how these changes were tested. If you've added new features, have you added unit tests?

## Docs
Describe where this code is documented. If it changes a documented interface, have the docs been updated?

### CODE REVIEW

File: bin/citrea/tests/bitcoin_e2e/config/rollup.rs [hunk 1]
```diff
@@ -30,6 +30,7 @@ pub fn default_rollup_config() -> RollupConfig {
             node_password: String::from("password"),
             network: bitcoin::Network::Regtest,
             da_private_key: None,
+            tx_backup_dir: "../../resources/bitcoin/inscription_txs".to_string(),
```
Comment (kpp) [inline bin/citrea/tests/bitcoin_e2e/config/rollup.rs:33]: I doubt this is the best approach for tests

Comment (eyusufatik) [reply bin/citrea/tests/bitcoin_e2e/config/rollup.rs:33]: why

Comment (eyusufatik) [reply bin/citrea/tests/bitcoin_e2e/config/rollup.rs:33]: discssed in dms

File: bin/citrea/tests/bitcoin_e2e/utils.rs [hunk 2]
```diff
@@ -138,3 +138,12 @@ pub fn tail_file(path: &Path, lines: usize) -> Result<()> {
...
     Ok(())
 }
+
+pub fn get_tx_backup_dir() -> PathBuf {
+    let workspace_root = get_workspace_root();
+    let mut path = workspace_root.to_path_buf();
+    path.push("resources");
+    path.push("bitcoin");
+    path.push("inscription_txs");
+    path
```
Comment (kpp) [inline bin/citrea/tests/bitcoin_e2e/utils.rs:148]: unwrap to_string() here

**Review by kpp — APPROVED** [review]

**Pull request merged**